src/common/GPOWrapper/GPOWrapper.cpp

```diff
@@ -176,4 +176,12 @@ namespace winrt::PowerToys::GPOWrapper::implementation
     {
         return static_cast<GpoRuleConfigured>(powertoys_gpo::getAllowedAdvancedPasteOnlineAIModelsValue());
     }
+    /* winrt::hstring GPOWrapper::GetConfiguredMwbPolicyDefinedIpMappingRules()
+    {
+        // Assuming powertoys_gpo::getConfiguredMwbPolicyDefinedIpMappingRules() returns a std::wstring
+        std::wstring rules = powertoys_gpo::getConfiguredMwbPolicyDefinedIpMappingRules();
+
+        // Convert std::wstring to winrt::hstring
+        return to_hstring(rules.c_str());
+    } */
```

Comment:
@jaimecbernardo 
I need your help: If I add this code, then the build fails. Error message is: Do not use C-Style casts.
My problem ist that the casts are happening within the WinRT library. How can we fix this?

Reply:
For now this problem is not blocking anymore.